BigCommerce/Integration/01_ManageBigCommerceConnection.md

```diff
@@ -0,0 +1,195 @@
+Additionally, if you have worked with *BigCommerce* before starting with the *Actindo Core1 Platform*, you can define the date or ID, from which existing orders should be imported.
+
+The *Offers* setting is only relevant for the first initial import of products from *BigCommerce*. You can specify that only those attributes are to be imported, that are mapped to the offer by the ETL data mapping. This allows you to maintain only certain attributes.
+
+
+#### Prerequisites
+
+- In the *Actindo Core1 Platform*, you are permitted to create connections in *Omni-Channel*.
+- A *BigCommerce* connection has been created.
+
+#### Procedure
+
+*Ommi-Channel > Settings > Tab CONNECTIONS*
+
+![Omni-Channel connections](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/OmniChannelConnectionCreated.png "[Omni-Channel connections]")
+
+ 1. Click the *BigCommerce* connection you want to edit in the list of connections.   
+   The *Credentials* and the *Settings* tab are displayed in the *Edit connection* view. By default, the *Credentials* tab is selected.
+
+    ![Connection credentials](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/ConnectionCredentials.png "[Connection credentials]")
+
+2. Click the *Settings* tab.
+
+3. Click the *Variant option type* entry.   
+   The *Variant option type used for variants* dropdown is displayed.
+
+    ![Variant option type](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/VariantOptionType.png "[Variant option type]")
+
+4. Select the *Variant option type* you want to use.
+   The following option types are available:
+   | Dropdown | Meaning   
+   |-----|-------   
+   |Dropdown| An offer variant is selected in a dropdown
+   |Rectangles | An offer variant is selected in a small rectangle, that contains the offer variant value. For example:![Rectangle](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/VariantOptionRectangle2.png "[Rectangle]")
+    |Radio buttons | An offer variant is selected with a radio button.   
+
+5. Click the *Order filters* option.   
+   A list of order status is displayed.
+
+   ![Order filters](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/OrderFilters.png "[Order filters]")  
+
+6. Enable the toggles at the order statuses you want to import to the *Actindo Core1 Platform*. For detailed information on the *BigCommerce* order statuses, see the *BigCommerce* documentation under [https://developer.bigcommerce.com/docs/rest-management/orders/order-status#get-all-order-statuses](https://developer.bigcommerce.com/docs/rest-management/orders/order-status#get-all-order-statuses).
+
+7. If desired, enter the first order ID you want to import in the *Import only orders with an ID as of* field. This option is only relevant if you have orders in *BigCommerce* that should not be imported to *Actindo* in general.
```

Comment:
Im UI steht "Import only orders with an ID greater than"... Stimmt das? Absichtlich gemacht? (as of ist aber schöner, gebe ich zu)